Documentation/PackedAssetBuilds.md

```diff
@@ -0,0 +1,122 @@
+# Packaged MultiplayerSample Builds
+To make relocatable client and server builds for the MultiplayerSample, we recommend making packaged builds. These package builds will contain the Game or Server Launcher and the bundled assets needed to run the launcher outside of the developer environment.
+
+You can make both release packaged builds or profile packaged builds, see https://www.o3de.org/docs/user-guide/packaging/windows-release-builds/.
+
+The guide below covers how to make profile packaged builds which are very useful for early sharing and play testing.
+
+## Windows Profile PAK Setup
+
+
+### Install WWise
+Go to https://www.audiokinetic.com/download/, create a login, log in, and download the installer. WWise is needed to process and package audio assets in the project.
+
+![WWise installer options](Media/wwise_installer_options.png)
+
+Inside the installer select the version to use.  Install version **2021.1.11.7933**, select both Authoring and SDK, Microsoft platform.
+
+![WWise version selection](Media/wwise_installer_version_selection.png)
+
+> REBOOT (or logout / login). Otherwise, the environment settings won't get picked up for any builds in Visual Studio. They will only apply to command-line builds, and only for any command-line windows that have been opened after the installer finishes.
+
+### Build profile build and process assets
+
+Build and run MPS as per the [README.md](../README.md) and ensure all assets are built.
+
+### Test the profile build
+
+* Open the game in editor
+    * load `NewStarBase` level
+    * Verify that game can launch and connect to local server from editor
+* Validate local game launcher can connect to local server
+
+### Build AssetBuilder
+
+You will need to build the [AssetBundler](https://www.o3de.org/docs/user-guide/packaging/asset-bundler/overview/) tool if not built.
+
+For example:
+```shell
+cmake --build build\windows --target AssetBundler --config profile -- /m /nologo
+```
+
+### Build monolithic game
+
+Build a second version of the executables as monolithic pak builds.
+
+``` shell
+// Create build files for a monolithic build that also disables all user/project registry settings overrides
```

Comment:
```suggestion
# Create build files for a monolithic build that also disables all user/project registry settings overrides
```
using `#` comments in a `shell` code block leads to better syntax highlighting:

![image](https://user-images.githubusercontent.com/34254888/231599230-a01e2516-c199-4420-9f11-95ea07b40406.png)